sdk/template/aztemplate/version.go

```diff
@@ -0,0 +1,15 @@
+//go:build go1.17
+// +build go1.17
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcore
+
+const (
+	// UserAgent is the string to be used in the user agent string when making requests.
+	UserAgent = "aztemplate/" + Version
+
+	// Version is the semantic version (see http://semver.org) of this module.
+	Version = "v0.1.0"
```

Comment:
Setting aside the "v0" conversation for now until we meet. Shouldn't this match the version in the changelog and be "v0.1.0-beta.1"?

Reply:
Good catch, fixed